extensions-core/avro-extensions/pom.xml

```diff
@@ -160,6 +160,10 @@
           <groupId>org.codehaus.jackson</groupId>
           <artifactId>jackson-mapper-asl</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
These exclusions needed to be performed since `AvroStreamInputRowParserTest` was unable to locate the `asMap` method of `com.google.common.collect.Maps` in the version of guava that shipped with these Pig dependencies.

@ssanthanam185, it turns out that we needed the Pig dependency for another test class, so I excluded `com.google.guava` from the Pig dependencies altogether. Thank you SO much for your help!